pkg/3scale/amp/operator/apicast_reconciler.go

```diff
@@ -768,6 +768,22 @@ func (r *ApicastReconciler) getSecretUIDs(ctx context.Context) (map[string]strin
 		}
 	}
 
+	// system-database
+
+	if r.apiManager.IsSystemDatabaseTLSEnabled() {
```

Comment:
All of the above seems apicast related only, why have the system/zync db TLS stuff in this file?